### PR TITLE
Support for new language API.

### PR DESCRIPTION
**Pull request by nikomakela**
PT-897. Pull request https://github.com/City-of-Helsinki/palvelutarjotin/pull/131 for API is needed for this pull request!

### CODE REVIEW

File: src/domain/occurrence/occurrenceGroupInfo/OccurrenceGroupInfo.tsx [hunk 1]
```diff
@@ -10,7 +10,8 @@ interface Props {
 const OccurrenceGroupInfo: React.FC<Props> = ({ occurrence }) => {
   const { t } = useTranslation();
 
-  const languages = occurrence.languages.map((language) => language.id);
+  const languages =
```
Comment (Sartonon) [inline src/domain/occurrence/occurrenceGroupInfo/OccurrenceGroupInfo.tsx:13]: According to type `|| []` seems to be redundant:

```
  const languages = occurrence.languages.edges.map(
    (edge) => edge?.node?.id ?? ''
  );
```

**Review by Sartonon — APPROVED** [review]
1 minor comment but approved

File: src/domain/occurrence/occurrenceGroupInfo/OccurrenceGroupInfo.tsx (hunk 1 above)
Comment (nikomakela) [reply src/domain/occurrence/occurrenceGroupInfo/OccurrenceGroupInfo.tsx:13]: Fixed as suggested.

**Review by Sartonon — APPROVED** [review]

**Comment by nikomakela** [comment]
Before that tiny fix this PR somehow passed the acceptance test. Now it fails, but it really should fail because the API is not yet deployed. Let's run the acceptance test again when the API is shipped.

**Comment by nikomakela** [comment]
> Before that tiny fix this PR somehow passed the acceptance test. Now it fails, but it really should fail because the API is not yet deployed. Let's run the acceptance test again when the API is shipped.

It get fixed by rerun already 🤔

**Pull request merged**